### DISCUSSION TITLE
auto bed levelling

### DISCUSSION THREAD

**Discussion by dietermeijer** [mriscoc/Ender3V2S1#1111] [General]
I have a CR touch and for some reason it does not go to auto bed levelling like it did before using a previous firmware . i downloaded the firmware with bl touch and also the script on my slicing software has auto bed levelling in it ? what have i done wrong?

**Comment by mriscoc** [comment]
I can only guess because your question is missing information.  but maybe you have flashed a UBL version and you are using ABL gcode, so you need to change the G29 to G29 P1.
